pkg/templateservicebroker/brokerapi/templateservicebroker/catalog.go

```diff
@@ -0,0 +1,48 @@
+package templateservicebroker
+
+import (
+	"net/http"
+	"strings"
+
+	templateapi "github.com/openshift/origin/pkg/template/api"
+	"github.com/openshift/origin/pkg/templateservicebroker/brokerapi"
+	"k8s.io/kubernetes/pkg/api"
+)
+
+func serviceFromTemplate(template *templateapi.Template) *brokerapi.Service {
+	metadata := make(map[string]interface{})
+	if displayName, ok := template.Annotations["openshift.io/display-name"]; ok {
+		metadata["displayName"] = displayName
+	}
+	// TODO: metadata[imageUrl"]
+	// TODO: metadata[longDescription"]
+	// TODO: metadata[providerDisplayName"]
+	// TODO: metadata[documentationUrl"]
+	// TODO: metadata[supportUrl"]
+	// TODO: list template parameters (https://github.com/openservicebrokerapi/servicebroker/issues/59)
+
+	return &brokerapi.Service{
+		Name:        template.Name,
+		ID:          string(template.UID),
+		Description: template.Annotations["description"],
+		Tags:        strings.Split(template.Annotations["tags"], ","),
+		Bindable:    true,
+		Metadata:    metadata,
+		Plans:       plans,
+	}
+}
+
+func (b *Broker) Catalog() *brokerapi.Response {
+	// TODO: support templates in other namespaces or make this configurable?
```

Comment:
yeah this will need to be configurable (initially via the master-config.yaml i suppose and then later via a standalone config file when we start running the broker as its own microservice)